bundles/org.openhab.transform.exec/src/main/java/org/openhab/transform/exec/internal/ExecTransformationService.java

```diff
@@ -35,6 +36,7 @@
 @NonNullByDefault
 @Component(property = { "openhab.transform=EXEC" })
 public class ExecTransformationService implements TransformationService {
+    private static final Pattern SPLIT_ON_SPACE = Pattern.compile("(['])((?:\\\\\\1|.)+?)\\1|([^\\s']+)");
```

Comment:
Are you really sure you need 6 consecutive backslash ?

Reply:
The solution from @J-N-K was copied directly. I mainly looked at the single quotes without line breaks. 
And that looked oke. I cannot tell if there is a specific usecase that needs these backslashes. Maybe @J-N-K  can comment.

Reply:
@J-N-K : please comment, I would like to merge this PR

Reply:
@lsiepel : Can you point me to the original code from @J-N-K ?
We have to conclude here ;)

Reply:
https://github.com/J-N-K/openhab-addons/pull/22/files